src/Microsoft.DotNet.Interactive.PowerShell/Commands/OutDisplayCommand.cs

```diff
@@ -54,7 +55,21 @@ public sealed class OutDisplayCommand : PSCmdlet
         /// </summary>
         protected override void ProcessRecord()
         {
+            
             object obj = InputObject is PSObject psObject ? psObject.BaseObject : InputObject;
+
+            // If object is PSCustomObject, convert to PSObject to find the properties to display
+            if (InputObject is PSObject && ((PSObject)InputObject).BaseObject is PSCustomObject)
+            {
+                Dictionary<string, object> table = new Dictionary<string, object>();
+                var props = ((PSObject)InputObject).Properties;
```

Comment:
With the above change, you can simplify this to:

```suggestion
                var props = psObj.Properties;
```